Mods/Editor/Src/Editor.h

```diff
@@ -145,10 +146,12 @@ class Editor : public IPluginInterface {
 private:
     ZEntityRef m_Camera;
     ZEntityRef m_CameraRT;
+    bool m_raycastLogging; // Mainly used for the raycasting logs
 
     bool m_CameraActive = false;
     ZEntityRef m_OriginalCam;
 
+    ZEntityRef s_SelectedEntity;
```

Comment:
Should use `m_` prefix for member variables.

Reply:
I've made it local now - not too sure why I made it a member